### PR TITLE
index.js entry file (Driver) is being flagged by the linter. Message …

### PR DESCRIPTION
**Pull request by hj9420**
…./src/index.js

Linter Error during compile. Figure I can make the small fix.

./src/index.js
  Line 11:  Extra semicolon  semi

### CODE REVIEW

**Comment by 18888651793** [comment]
这是来自QQ邮箱的假期自动回复邮件。 您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by hj9420** [comment]
Before:
image

After:
image

**Comment by bryersvan** [comment]
How would a coin creator begin a pull request to begin mining coins on Solana blockchain?

File: src/index.js [hunk 1]
```diff
@@ -8,6 +8,6 @@ import registerServiceWorker from './registerServiceWorker'
 // import { unregister as unregisterServiceWorker } from './registerServiceWorker';
 
 ReactDOM.render(<App />, document.getElementById('root'))
-registerServiceWorker();
```
Comment (blmitkc) [inline src/index.js:11]:
```

**Review by blmitkc — APPROVED** [review]

**Pull request closed without merging**